packages/phone-number-privacy/combiner/src/signing/get-threshold-signature.ts

```diff
@@ -73,6 +73,8 @@ async function requestSignatures(request: Request, response: Response) {
   })
   obs.observe({ entryTypes: ['measure'], buffered: true })
 
+  request.headers.keyVersion = config.keyVersions.phoneNumberPrivacy.toString()
```

Comment:
Convention for HTTP(S) headers is to name them in Hyphen-Delimited-Camel-Case. Additionally, it's important to be reasonably confident that the name will not collide with a header provided by any other application.
```suggestion
  request.headers['Odis-Key-Version'] = config.keyVersions.phoneNumberPrivacy.toString()
```